unbxdAutosuggest.js

```diff
@@ -1281,6 +1329,25 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, undefined) {
 			}
 
 		}
+		, sortSuggestionsBylength: function () {
+			this.currentResults.SORTED_SUGGESTIONS = this.currentResults.KEYWORD_SUGGESTION.concat(this.currentResults.TOP_SEARCH_QUERIES);
+			this.currentResults.SORTED_SUGGESTIONS.sort(function (a, b) {
+				if (a.autosuggest.length - b.autosuggest.length == 0) {
```

Comment:
use `===` for being on the safer side.

Below two are different:
`"0" == 0` => returns true;
`"0" === 0` => returns false;

`===` returns true if both operands are of the same type and contain the same value.

Reply:
How does `a.autosuggest.length - b.autosuggest.length` return `"0"` ?

Reply:
@ganeshparsads Done